components/shapediver/parameter/select/SelectImageDropDownComponent.tsx

```diff
@@ -0,0 +1,77 @@
+import {Group, Image, Select, SelectProps, Text} from "@mantine/core";
+import React from "react";
+import {SelectComponentProps} from "./SelectComponent";
+
+/**
+ * Custom data type for select options with image and description
+ */
+interface CustomSelectItem {
+	value: string;
+	label: string;
+	description?: string;
+	imageUrl?: string;
+	width?: number;
```

Comment:
@LemToUp please see this comment: https://github.com/shapediver/AppBuilderShared/pull/77/files#r1992015404

Reply:
Replaced https://github.com/shapediver/AppBuilderShared/pull/77/commits/7c57aeba8fecb6c50269ca31ee057b7b409d8113